packages/web/pages/quests.tsx

```diff
@@ -51,6 +52,11 @@ const QuestsPage: React.FC<Props> = () => {
 
   return (
     <PageContainer>
+      <HeadComponent
+        title="Metagame's Quests"
+        description="Metagame is a massive online coordination game. Metagame has some epic quests going on!"
```

Comment:
Could we keep the first sentence of the description consistent across all pages? 
I see these three different occurrences which can all be replaced with one of them:

1. A Massive Online Coordination Game!
2. Metagame is a Massive Online Coordination Game!
3. Metagame is a massive online coordination game.

Reply:
resolved by 787998b